### ISSUE TITLE
simplejson.errors.JSONDecodeError

### ISSUE THREAD

**Issue by richarddwang** [neptune-ai/neptune-client#280]
This didn't happen with the same code, and I am using the fastai neptune callback
```
Traceback (most recent call last):
  File "finetune.py", line 116, in <module>
    neptune.init(project_qualified_name='natsume/electra-glue')
  File "/home/yisiang/miniconda3/envs/ml/lib/python3.7/site-packages/neptune/__init__.py", line 148, in init
    backend = HostedNeptuneBackend(api_token, proxies)
  File "/home/yisiang/miniconda3/envs/ml/lib/python3.7/site-packages/neptune/utils.py", line 210, in wrapper
    return func(*args, **kwargs)
  File "/home/yisiang/miniconda3/envs/ml/lib/python3.7/site-packages/neptune/internal/backends/hosted_neptune_backend.py", line 91, in __init__
    self._client_config = self._create_client_config(self.credentials.api_token, backend_client)
  File "/home/yisiang/miniconda3/envs/ml/lib/python3.7/site-packages/neptune/utils.py", line 210, in wrapper
    return func(*args, **kwargs)
  File "/home/yisiang/miniconda3/envs/ml/lib/python3.7/site-packages/neptune/internal/backends/hosted_neptune_backend.py", line 930, in _create_client_config
    config = backend_client.api.getClientConfig(X_Neptune_Api_Token=api_token).response().result
  File "/home/yisiang/miniconda3/envs/ml/lib/python3.7/site-packages/bravado/http_future.py", line 200, in response
    swagger_result = self._get_swagger_result(incoming_response)
  File "/home/yisiang/miniconda3/envs/ml/lib/python3.7/site-packages/bravado/http_future.py", line 124, in wrapper
    return func(self, *args, **kwargs)
  File "/home/yisiang/miniconda3/envs/ml/lib/python3.7/site-packages/bravado/http_future.py", line 303, in _get_swagger_result
    self.request_config.response_callbacks,
  File "/home/yisiang/miniconda3/envs/ml/lib/python3.7/site-packages/bravado/http_future.py", line 337, in unmarshal_response
    op=operation,
  File "/home/yisiang/miniconda3/envs/ml/lib/python3.7/site-packages/bravado/http_future.py", line 374, in unmarshal_response_inner
    content_value = response.json()
  File "/home/yisiang/miniconda3/envs/ml/lib/python3.7/site-packages/bravado/requests_client.py", line 160, in json
    return self._delegate.json(**kwargs)
  File "/home/yisiang/miniconda3/envs/ml/lib/python3.7/site-packages/requests/models.py", line 898, in json
    return complexjson.loads(self.text, **kwargs)
  File "/home/yisiang/miniconda3/envs/ml/lib/python3.7/site-packages/simplejson/__init__.py", line 525, in loads
    return _default_decoder.decode(s)
  File "/home/yisiang/miniconda3/envs/ml/lib/python3.7/site-packages/simplejson/decoder.py", line 370, in decode
    obj, end = self.raw_decode(s)
  File "/home/yisiang/miniconda3/envs/ml/lib/python3.7/site-packages/simplejson/decoder.py", line 400, in raw_decode
    return self.scan_once(s, idx=_w(s, idx).end())
simplejson.errors.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**Comment by PiotrJander** [comment]
@richarddwang thanks for reporting this, we now know that this is due to a bug in a library our client uses https://github.com/Yelp/bravado/issues/222 . We will find a workaround and let you know when this is fixed.

**Comment by PiotrJander** [comment]
Anyone who is seeing this as well, please upvote the issue.

**Comment by PiotrJander** [comment]
@richarddwang do you encounter this problem consistently or only occasionally?

**Comment by richarddwang** [comment]
I start run after run in a program, and there is chance the program run for 6 hr with some completed runs and run into the issue.

**Comment by richarddwang** [comment]
It seems that something has been fixed, and  I am not experiencing this issue for a while. It should be good. Thanks !

**Comment by HoltSpalding** [comment]
I'm running into this issue

**Comment by shnela** [comment]
Hi @HoltSpalding 

Could you please send us a tracebkack, and tell how can we reproduce this error?

Best
Jakub

**Comment by ben-heil** [comment]
I'm running into this error while using Bravado version 11.0.3 while trying to sync offline runs

```
File "~/anaconda3/envs/saged/bin/neptune", line 10, in <module>
    sys.exit(main())
  File "~/anaconda3/envs/saged/lib/python3.7/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "~/anaconda3/envs/saged/lib/python3.7/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "~/anaconda3/envs/saged/lib/python3.7/site-packages/click/core.py", line 1668, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "~/anaconda3/envs/saged/lib/python3.7/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "~/anaconda3/envs/saged/lib/python3.7/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "~/anaconda3/envs/saged/lib/python3.7/site-packages/neptune/new/sync.py", line 418, in sync
    sync_all_runs(path, project_name)
  File "~/anaconda3/envs/saged/lib/python3.7/site-packages/neptune/new/sync.py", line 320, in sync_all_runs
    sync_offline_runs(base_path, project_name, offline_runs_ids)
  File "~/anaconda3/envs/saged/lib/python3.7/site-packages/neptune/new/sync.py", line 298, in sync_offline_runs
    project = get_project(project_name)
  File "~/anaconda3/envs/saged/lib/python3.7/site-packages/neptune/new/sync.py", line 89, in get_project
    return backend.get_project(project_name)
  File "~/anaconda3/envs/saged/lib/python3.7/site-packages/neptune/new/internal/backends/utils.py", line 61, in wrapper
    return func(*args, **kwargs)
  File "~/anaconda3/envs/saged/lib/python3.7/site-packages/neptune/new/internal/backends/hosted_neptune_backend.py", line 182, in get_project
    **self.DEFAULT_REQUEST_KWARGS,
  File "~/anaconda3/envs/saged/lib/python3.7/site-packages/bravado/http_future.py", line 200, in response
    swagger_result = self._get_swagger_result(incoming_response)
  File "~/anaconda3/envs/saged/lib/python3.7/site-packages/bravado/http_future.py", line 124, in wrapper
    return func(self, *args, **kwargs)
  File "~/anaconda3/envs/saged/lib/python3.7/site-packages/bravado/http_future.py", line 303, in _get_swagger_result
    self.request_config.response_callbacks,
  File "~/anaconda3/envs/saged/lib/python3.7/site-packages/bravado/http_future.py", line 337, in unmarshal_response
    op=operation,
  File "~/anaconda3/envs/saged/lib/python3.7/site-packages/bravado/http_future.py", line 380, in unmarshal_response_inner
    content_value = response.json()
  File "~/anaconda3/envs/saged/lib/python3.7/site-packages/bravado/requests_client.py", line 160, in json
    return self._delegate.json(**kwargs)
  File "~/anaconda3/envs/saged/lib/python3.7/site-packages/requests/models.py", line 898, in json
    return complexjson.loads(self.text, **kwargs)
  File "~/anaconda3/envs/saged/lib/python3.7/site-packages/simplejson/__init__.py", line 525, in loads
    return _default_decoder.decode(s)
  File "~/anaconda3/envs/saged/lib/python3.7/site-packages/simplejson/decoder.py", line 370, in decode
    obj, end = self.raw_decode(s)
  File "~/anaconda3/envs/saged/lib/python3.7/site-packages/simplejson/decoder.py", line 400, in raw_decode
    return self.scan_once(s, idx=_w(s, idx).end())
simplejson.errors.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**Comment by PiotrJander** [comment]
Hi @ben-heil , I assume that your problem is not fixed by the issue you previously reported https://github.com/neptune-ai/neptune-client/issues/282 

Your problem looks similar to https://github.com/neptune-ai/neptune-client/issues/497 . Could you take a look there and see if the solution works for you?

**Comment by ben-heil** [comment]
Hi @PiotrJander, I saw that issue, but I run into the error when running `neptune sync` from command line instead of `neptune_init` from the python interface so I thought it didn't apply. I believed that the issue was something other than the project name, since I get a different error if I use a project name that doesn't exist. However, I just tried using `ben-heil/saged` instead of just `saged` for the project name and that seems to have fixed it. Thanks!
